WORKSPACE

```diff
@@ -118,6 +118,7 @@ haskell_nixpkgs_packageset(
     base_attribute_path = "haskellPackages",
     nix_file = "//tests:ghc.nix",
     repositories = {"nixpkgs": "@nixpkgs"},
+    nix_build_extra_args = ["-j", "1"],
```

Comment:
I don't know, because most of the time `nixpkgs_package` would be called only a handful of times, but on packages with a relatively big closure, so getting rid of the nix-level parallelism would have a huge impact on the build of these dependencies

Reply:
Good point. I wonder what should be the "right" default. One will slow down build but the other may generate really big load on the machine and high memory usage.

Reply:
I agree with @guibou. In general there should always be just a single level of parallelism. Only introduce multiple levels in exceptional circumstances when you know what you're doing, and start with top-level parallelism first. In fact in Nix, the defaults are `nix.buildCores = 1` and `nix.maxJobs = 1`. The `-j 1` here is morally like `nix.buildCores` and `--loading_phase_threads` like `nix.maxJobs`.